mailpoet/lib/Form/Widget.php

```diff
@@ -2,6 +2,9 @@
 
 namespace MailPoet\Form;
 
+if (!defined('ABSPATH')) exit;
+
+
```

Comment:
Similarly to https://github.com/mailpoet/mailpoet/pull/6358, this shouldn't be included in the file, as it's automatically added by the build process.